src/components/create.js

```diff
@@ -22,7 +22,7 @@ const CreateBook = () => {
         },
         category,
       };
-      dispatch(addBook(newBook));
+      dispatch(addBookToAPI(newBook));
```

Comment:
- [ ] I am unable to verify if the book is correctly added to the API because when I reload the page no book is fetched and I also get this error message in the console.

  ![image](https://user-images.githubusercontent.com/54848363/210333511-30ab23ab-3f58-4555-a15c-f1592bd0dc53.png)